cfg.go

```diff
@@ -16,6 +18,7 @@ const (
 
 	defaultConfigEnv = "default"
 	localConfigEnv   = "local"
+	envVarConfigFile = "environment-variables"
```

Comment:
Maybe `custom-environmen-variables`? -> https://github.com/lorenwest/node-config/wiki/Environment-Variables#custom-environment-variables

Reply:
I think an smaller name is better 🙊